micromamba/src/shell.cpp

```diff
@@ -86,16 +169,18 @@ namespace
                 config.load();
                 shell_init(
                     consolidate_shell(config.at("shell_type").compute().value<std::string>()),
-                    config.at("shell_prefix").compute().value<std::string>()
+                    Context::instance().prefix_params.root_prefix
```

Comment:
I'm a bit lost here, why do we replace `shell_prefix` with the `root_prefix`? Is it because we know that we use it in here deterministically? (same for the other replacements below).

Reply:
It was always the root_prefix 🙃 
It was that way because the same CLI option was used as a catch all root/prefix.